### PR TITLE
need tools combinations

### PR DESCRIPTION
**Pull request by Bioninbo**
Hello,
I am writing a pipeline for ATAC-Seq. Most of my processes require single tool containers, but for some I need multi-tools. It would be really great if you could make these containers so I can package my pipeline in conda and singularity. I could not find mulled containers fitting these requirements.
Best,
Jerome

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: combinations/hash.tsv [hunk 1]
```diff
@@ -251,4 +251,10 @@ changeo=1.2.0,igblast=1.17.1,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bas
 cnvkit=0.9.9,samtools=1.15.1
 deeptools=3.5.1,samtools=1.15.1
 r-recetox-aplcms=0.9.4,r-base=4.1.0,r-arrow=4.0.1,r-dplyr=1.0.7	quay.io/bioconda/base-glibc-debian-bash:latest	0
+r-base=4.1.3--h06d3f91_1,r-magrittr=2.0.3--r41h06615bd_0,r-dplyr=1.0.9--r41h7525677_0,r-purrr=0.3.4--r41hcfec24a_1,r-ggplot2=3.3.5--r41hc72bb7e_0,r-data.table=1.14.2--r41hcfec24a_0
+samtools=1.15.1--h1170115_0,bedtools=2.30.0--h468198e_3,perl=5.32.1--2_h7f98852_perl5
+skewer=0.2.2--h2d50403_2,pigz=2.6--h27826a3_0
+r-base=4.1.3,bioconductor-chipseeker=1.30.0,r-magrittr=2.0.3,bioconductor-genomicfeatures=1.46.1,bioconductor-clusterprofiler=4.2.0,bioconductor-annotationdbi=1.56.1,r-purrr=0.3.4,r-ggplot2=3.3.5
+r-base=4.1.3,r-ggplot2=3.3.5,r-magrittr=2.0.3,r-gridextra=2.3,r-rcolorbrewer=1.1_3,r-data.table=1.14.2
+r-base=4.1.3,bioconductor-diffbind=3.4.11,r-sleuth=0.30.0,r-ggplot2=3.3.5,r-magrittr=2.0.3,r-openxlsx=4.2.5
```
Comment (drpatelh) [inline combinations/hash.tsv:259]:
```suggestion
r-base=4.1.3,r-magrittr=2.0.3,r-dplyr=1.0.9,r-purrr=0.3.4,r-ggplot2=3.3.5,r-data.table=1.14.2
samtools=1.15.1,bedtools=2.30.0,perl=5.32.1
skewer=0.2.2,pigz=2.6
r-base=4.1.3,bioconductor-chipseeker=1.30.0,r-magrittr=2.0.3,bioconductor-genomicfeatures=1.46.1,bioconductor-clusterprofiler=4.2.0,bioconductor-annotationdbi=1.56.1,r-purrr=0.3.4,r-ggplot2=3.3.5
r-base=4.1.3,r-ggplot2=3.3.5,r-magrittr=2.0.3,r-gridextra=2.3,r-rcolorbrewer=1.1_3,r-data.table=1.14.2
r-base=4.1.3,bioconductor-diffbind=3.4.11,r-sleuth=0.30.0,r-ggplot2=3.3.5,r-magrittr=2.0.3,r-openxlsx=4.2.5
```

Comment (drpatelh) [reply combinations/hash.tsv:259]: Hi @Bioninbo ! I suspect you don't need to provide build ids here. Also, not sure what the etiquette is here in terms of building multiple containers on the same PR. Maybe @bgruening can clarify.

Comment (Bioninbo) [reply combinations/hash.tsv]: Thank you @drpatelh ! I have removed the build ids in a new commit.

Comment (bgruening) [reply combinations/hash.tsv]: Multiple containers should be ok in one PR.

**Pull request merged**